tests/integration/models/reddit/test_emoji.py

```diff
@@ -0,0 +1,28 @@
+from praw.models import Emoji
```

Comment:
Can you add a line break between mock and ... import to keep the import grouping distinguished? E.g., system imports first (there are none), then third party, then relative?